test/mock-vendor.go

```diff
@@ -5,7 +5,5 @@ import "github.com/golang/mock/mockgen/model"
 // This assignment exists solely for the purpose of convincing godep
 // to vendor github.com/golang/mock/mockgen/model as gomock will fail
 // to generate code if it doesn't exist in the users GOPATH but isn't
-// actually imported by any boulder or gomock code. The variable name
-// is chosen so that it is unlikely to clash with anything else in this
-// package.
-var _ignore = model.Package{}
+// actually imported by any boulder or gomock code.
+var _ = model.Package{}
```

Comment:
We tried this same thing with godep and it broke things at the time, does this actually work with go mod?

Reply:
Yep, tried it just now and it works with go mod vendor.